CoverMyDotNet.Samples/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Configuration;
+using System.Collections.Generic;
+using System.Linq;
+namespace CoverMyDotNet.Samples
+{
+	class MainClass
+	{
+		public static void MultipleRequests(Client client)
+		{
+			var tokens = new List<string>();
+			tokens.Add(client.CreateRequest(new RequestAttributes()
+			{
+				State = "OH",
+				Prescription = new PrescriptionAttributes()
+				{
+					DrugId = "094563"
+				},
+				Patient = new PatientAttributes()
+				{
+					FirstName = "John",
+					LastName = "test1",
+					DateOfBirth = "3/4/1992"
+				}
+			}).Tokens.First().Id);
+			tokens.Add(client.CreateRequest(new RequestAttributes()
+			{
+				State = "NY",
+				Prescription = new PrescriptionAttributes()
+				{
+					DrugId = "094563"
+				},
+				Patient = new PatientAttributes()
+				{
+					FirstName = "Mark",
+					LastName = "test3",
+					DateOfBirth = "3/4/1990"
+				}
+			}).Tokens.First().Id);
+			tokens.Add(client.CreateRequest(new RequestAttributes()
```

Comment:
I like what you did with varying the fields.  It also reveals more of what is supported.